### ISSUE TITLE
Portable version of NVDA cannot bring up its GUI in Windows 8 metro apps

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1800]
**Reported by mdcurran on 2011-09-17 06:05**
Metro apps in Windows8 are full-screen, and obviously have particular rules as to what can show other GUIs over the top or at least nock Windows out of the metro app in question in order to show the requested GUI.
When NVDA is installed (and has UIAccess privilages) bringing up the NVDA menu or a NVDA dialog while in a metro app knocks you out of the Metro app and on to the desktop first before bringing up the GUI. I think this is okay and is probably expected behaviour. 
However, if NVDA does not have uiAccess privilages, then bringing up the NVDA menu seems to fail, and then from then on you are no longer able to alt+tab out of the metro app. This seems to act in a similar way with NVDA dialogs as well.

**Comment by Adriani90** [comment]
@@michaelDCurran can you still see this issue in IE11?

**Comment by ehollig** [comment]
CC @bhavyashah

**Comment by larry801** [comment]
I tested this on Windows 8.1 Enterprise.
If I press NVDA+n under Metro app Windows Mail ,NVDA says "NVDA menu".
Then if I press up or down arrow, I cannot hear menu items.
Press NVDA+n again would raise an error.
Here is full log.
[nvda_portable_win8_1.log](https://github.com/nvaccess/nvda/files/2292243/nvda_portable_win8_1.log)

**Comment by Adriani90** [comment]
Support for Windows 8.1 ended completely on january 10, 2023, and full screen Windows 8.1 Metro Apps are discontinued or not developed anymore. Closing as abandoned, since I don't expect any updates regarding accessibility or further improvements.
